lighthouse-core/report/v2/renderer/category-renderer.js

```diff
@@ -336,6 +350,86 @@ class CategoryRenderer {
   }
 
   /**
+   * Render a screenshot filmstrip and metrics bars
+   * @param {!Array<!ReportRenderer.AuditJSON>} metricAudits
+   * @param {*} artifacts
+   * @return {?Element}
+   */
+  _renderPerfTimeline(metricAudits, artifacts) {
+    if (!artifacts)
+      return;
+    const tracePass = artifacts['traces'] ? artifacts['traces']['defaultPass'] : null;
+    if (!tracePass)
+      return;
+
+    const fmp = metricAudits.find(a => a.id === 'first-meaningful-paint').result;
+    if (!fmp || !fmp['extendedInfo'])
+      return;
+
+    const tti = metricAudits.find(a => a.id === 'time-to-interactive').result;
+    if (!tti || !tti['extendedInfo'])
+      return;
+
+    const navStart = fmp['extendedInfo']['value']['timestamps']['navStart'];
+    const markers = [
+      {
+        title: 'First contentful paint',
```

Comment:
I like the idea of having top level audits for all of the ones that have bars. But that can also wait until after IO.

Should we consolidate these two sections to remove duplicated info? We could put the score next to the bar and make the bar section expandable with help text. Move others without bars (e.g. EIL) to the bottom so the bars stay together.